docs/source/DeveloperDocumentation/TaskEngineWorkflowFeatures.md

```diff
@@ -464,3 +464,7 @@ The example below would result in a live stream where assets `source_1.m3u8`, `s
   }
 }
 ```
+
+## CONTINUOUS CAPTURE
+
+The Task Engine [VOD Capture](TaskEngineWorkflows.md#vod-capture) workflow supports the `continuous_capture` parameter which will begin capturing clips before the specified end time. By default it will begin capturing once 80% of the time has elapsed between the earliest start time of the specified clips and the latest end time. This can be overriden by setting the `run_at` parameter.
```

Comment:
I would mention that if the clip is under 60 seconds the start time will be used as the run_at time (job_request.rb - line 87)

```
 elsif end_time - start_time < 60
      start_time
    else
 ```